src/briefcase/console.py

```diff
@@ -118,6 +162,67 @@ def error(self, message="", *, prefix="", markup=False):
         """Log message at error level; always included in output."""
         self._log(prefix=prefix, message=message, markup=markup, style="bold red")
 
+    def capture_stacktrace(self):
+        """Extract Rich stacktrace from exception while in except block."""
+        self.stacktrace = Traceback.extract(*sys.exc_info(), show_locals=True)
+
+    def save_log_to_file(self, command):
+        """Save the current application log to file."""
+        # only save log if a command ran and it errored or --save-log passed
+        if command is None or (not self.stacktrace and not command.save_log):
+            return
+
+        with command.input.wait_bar("Saving application log...", transient=True):
+            self.cleanup_log_files(command.logs_path)
+
+            if self.stacktrace:
+                self.rich_log_file_console.print()
+                self.rich_log_file_console.print(
+                    Traceback(
+                        trace=self.stacktrace,
+                        width=LOG_FILE_WIDTH,
+                        show_locals=True,
+                    )
+                )
+
+            now = datetime.now()
+            log_filepath = (
+                command.logs_path / f"briefcase_{now.strftime('%Y_%m_%d-%H_%M_%S')}.log"
+            )
+            rich_console.print()
+            try:
+                command.logs_path.mkdir(parents=True, exist_ok=True)
+                with open(
+                    log_filepath, mode="w", encoding="utf-8", errors="ignore"
+                ) as log_file:
+                    log_file.write(
```

Comment:
I guess Python defaults to 1252 on Windows and immediately started get upset about some of the characters that Rich was putting in the file. Setting the `encoding` was enough but figured setting `errors` would prioritize creating log files over quibbling about strange characters that probably aren't particularly relevant.

Reply:
Good point. Replaced with `backslashreplace`.